src/webgl/p5.Quat.js

```diff
@@ -0,0 +1,90 @@
+import p5 from '../core/main';
+
```

Comment:
If this file should be part of the public-facing documentation, can we also add module/submodule information? Example here for `p5.Vector`:

https://github.com/processing/p5.js/blob/eb61f7a260531d17930de3c987949558ce242d35/src/math/p5.Vector.js#L1-L5